src/org/usfirst/frc/team1529/robot/commands/HandIntakeCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team1529.robot.commands;
+
+import org.usfirst.frc.team1529.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class HandIntakeCommand extends Command {
+
+    public HandIntakeCommand() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	requires(Robot.kHandSubsystem);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	Robot.kHandSubsystem.Intake();
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished() {
+        return true;
+    }
+
+    // Called once after isFinished returns true
+    protected void end() {
```

Comment:
Might want to kill the motor when the command is done. Right now you are saying to go to the speed and never turn off.